tests/README.md

```diff
@@ -192,32 +192,10 @@ source tree. This directory is bind-mounted in the container and used as a
 local image cache.
 
 `Q5:`
-*Is there a way to speed up integration tests execution time?*
```

Comment:
Was the FAQ removed because it was inaccurate?

Reply:
Fixed